core/services/chainlink/config_general.go

```diff
@@ -95,13 +102,17 @@ func (o GeneralConfigOpts) New() (GeneralConfig, error) {
 		}
 	}
 
-	if o.SecretsString != "" {
-		err := o.parseSecrets()
+	for _, s := range o.SecretsStrings {
+		err := o.parseSecrets(s)
 		if err != nil {
 			return nil, err
 		}
 	}
 
+	if nil == o.Secrets.Database.AllowSimplePasswords {
+		o.Secrets.Database.AllowSimplePasswords = new(bool)
+	}
```

Comment:
I wonder if this should be wrapped up in a `setDefaults()` method for consistency?

Reply:
Yea, good call!